Sources/BraintreeShopperInsights/BTButtonOrder.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+/// The order or ranking in which payment buttons appear.
+/// - Warning: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTButtonOrder: Int {
```

Comment:
We can update this to a string enum and set the values based on our slack conversation